cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -684,10 +684,11 @@
     @Deprecated
     public static final String AUTH_SERVER_ADDRESS = "security.auth.server.address";
     /**
-     * Address that clients should use to communicate with the Authentication Server.
-     * Leave empty to use default, generated by the Authentication Server.
+     * Comma separated URL's that clients should use to communicate with the Authentication Server.
+     * Each URL should follow the format protocol://host:port. Leave empty to use the default URL generated by
+     * the Authentication Server.
      */
-    public static final String AUTH_SERVER_ANNOUNCE_ADDRESS = "security.auth.server.announce.address";
```

Comment:
You can't just change the configuration. The old configuration is being used in 3.5, 4.0 and 4.1